client/packages/invoices/src/InboundShipment/DetailView/AppBarButtons.tsx

```diff
@@ -57,13 +61,16 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   return (
     <AppBarButtonsPortal>
       <Grid container gap={1}>
-        <ButtonWithIcon
-          disabled={isDisabled}
-          label={t('button.add-item')}
-          Icon={<PlusCircleIcon />}
-          onClick={() => onAddItem(true)}
+        <AddButton
+          onAddItem={onAddItem}
+          requisitionId={data?.requisition?.id ?? ''}
+          invoiceId={data?.id ?? ''}
+          disable={isDisabled}
+          disableAddFromMasterListButton={
+            data?.status !== InvoiceNodeStatus.New
```

Comment:
Is there a reason why add from master list and add from internal order are only available in New status, but adding items is ok?
Not a major was just wondering why the logic is different for those? Particularly for inbound if you missed a line it would be good to be able to link to it?

Reply:
Not to sure, the disable from master list button was always like this. I'm assuming that it's because 'delivered' can be from an outbound -> inbound, if that's the case we can just disable by linked id